packages/lumx-react/src/components/text-field/TextField.stories.tsx

```diff
@@ -53,6 +53,7 @@ export const Clearable = {
     args: {
         value: 'Some value',
         clearButtonProps: { label: 'Clear' },
+        onClear: () => console.log('onClear called'),
```

Comment:
Can you use SB actions in argTypes instead ? It's a bit cleaner that way 
```
argTypes: { onClear: { action: true } },
```

Reply:
done ☑️